Slim/Player/Playlist.pm

```diff
@@ -52,7 +52,12 @@ sub shuffleType {
 	return 'none';
 }
 
-sub song {
+sub song { 
+	$log->is_warn && logBacktrace('deprecated!') ;
```

Comment:
I like the message you had before (and I can't find any more). Something like:

```perl
$log->warn("Slim::Player::Playlist::song() is deprecated, please use Slim::Player::Playlist::track() instead");
main::INFOLOG && $log->is_info && logBacktrace('Caller:');
```

This would always give us a warning, to raise the attention, without flooding the log file with backtraces. But upon request the user could increase the log level to figure out where the call is coming from.